tests/edgeless/connector.spec.ts

```diff
@@ -396,36 +336,75 @@ test('path #1', async ({ page }) => {
     { x: end[0], y: end[1] }
   );
 
-  start = await toViewCoord(page, [300, -100]);
-  end = await toViewCoord(page, [400, 0]);
+  start = await toViewCoord(page, [200, -100]);
+  end = await toViewCoord(page, [300, 0]);
   await addBasicRectShapeElement(
     page,
     { x: start[0], y: start[1] },
     { x: end[0], y: end[1] }
   );
 
   start = await toViewCoord(page, [50, 0]);
-  end = await toViewCoord(page, [350, 0]);
+  end = await toViewCoord(page, [250, 0]);
   await addBasicConnectorElement(
     page,
     { x: start[0], y: start[1] },
     { x: end[0], y: end[1] }
   );
+
+  await waitNextFrame(page);
   await assertConnectorPath(page, [
     [50, 0],
-    [50, -60],
-    [200, -60],
-    [200, 60],
-    [350, 60],
-    [350, 0],
+    [50, -20],
+    [150, -20],
+    [150, 20],
+    [250, 20],
+    [250, 0],
   ]);
 });
 
-test('path #2', async ({ page }) => {
-  await enterPlaygroundRoom(page);
-  await initEmptyEdgelessState(page);
-  await switchEditorMode(page);
-  await deleteAll(page);
+test('path #2, the top-right point is overlapped with the bottom-left point of another, and anchor from top to bottom of another', async ({
```

Comment:
Please migrate all test cases in this file according to 702bc472200dd0b587016026a489982737999b9f. Current writeup is too verbose. Also this kind of reassigning `let start, end` is unnecessary and should be avoided when possible.